src/components/profile/SecuritySection.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <BaseSection title="Security" subtitle="Manage the authentication credentials and security related options.">
+        <div class="space-y-6">
+            <div class="lg:flex lg:space-x-12 lg:space-y-0 space-y-4 w-full">
+                <div class="lg:w-1/2 w-full">
+                    <label class="input-label-tmp">Username</label>
+                    <input
+                        id="userName"
+                        v-model="accountUsername"
+                        type="text"
+                        class="w-full input-text-tmp"
+                        :class="errorBag.hasNested('username') ? 'input-text-error-tmp' : 'input-text-tmp'"
+                        :readonly="editMode"
+                    />
+                    <label v-if="errorBag.hasNested('username')" class="input-label-error-tmp">
+                        {{ errorBag.getNested("username") }}
+                    </label>
+                </div>
+                <div class="invisible lg:w-1/2 w-full" />
+            </div>
+            <div v-if="!editMode" class="lg:flex lg:space-x-12 lg:space-y-0 space-y-4 w-full">
+                <div class="lg:w-1/2 w-full relative">
+                    <label for="password" class="input-label-tmp">Password</label>
+                    <i
+                        :class="[isPasswordVisible() ? 'fa-eye-slash' : 'fa-eye']"
+                        class="fas absolute z-20 mt-2.5 mr-4 right-0 text-gray-500 cursor-pointer"
+                        @click="togglePassword"
+                    />
+                    <input
+                        id="password"
+                        v-model="accountPassword"
+                        :type="passwordFieldType"
```

Comment:
Not too sure about this, but I reckon it is because the browser caches the id. We use the exact same id on different pages which could lead to a problem as they should be completely unique. Might consider changing the id here and adjust the test cases if necessary.